bodhi/tests/server/test_utils.py

```diff
@@ -88,6 +88,21 @@ def test_sorted_builds(self):
         assert b1 == new, b1
         assert b2 == old, b2
 
+    def test_splitter(self):
+        splitlist = util.splitter(["build-0.1", "build-0.2"])
+        assert splitlist[0] == "build-0.1"
+        assert splitlist[1] == "build-0.2"
```

Comment:
For all three of these test blocks, you could replace the assertions with a one liner that also asserts that the length is 2:

```
self.assertEqual(splitlist, ['build-0.1', 'build-0.2'])
```